lib/kubeclient/common.rb

```diff
@@ -405,6 +405,17 @@ def proxy_url(kind, name, port, namespace = '')
       end
     end
 
+    def process_template(template)
+      ns_prefix = build_namespace_prefix(template[:metadata][:namespace])
+      @headers['Content-Type'] = 'application/json'
+      response = handle_exception do
+        rest_client[ns_prefix + 'processedtemplates']
+        .post(template.to_h.to_json, @headers)
```

Comment:
Question: I dont understand how this is different from `create_template`. They are both using `post` on the received template hash. Can you please explain how this works?